src/proto/beats.proto

```diff
@@ -10,10 +11,23 @@ enum Source {
   USER = 2;
 }
 
+enum Type {
```

Comment:
Please use enum class and PascalCase for members

Reply:
This is a .proto file, not .cpp. I'm not sure if enum classes are supported.

Reply:
Oops, sorry

Reply:
Yep, proto buffer has his own syntax.